pt/jenkins/pt-package-testing.groovy

```diff
@@ -0,0 +1,359 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+product_action_playbooks = [
+    pt3: [
+        install: 'pt.yml',
+        upgrade: 'pt_upgrade.yml',
+        pt_with_products: 'pt_with_products.yml'
+    ]
+]
+
+setup_centos_package_tests = { ->
+    sh '''
+        sudo yum install -y epel-release
+        sudo yum -y update
+        sudo yum install -y ansible
+    '''
+}
+
+setup_centos8_package_tests = { ->
+    sh '''
+        sed -i 's/mirror\.centos\.org/vault\.centos\.org/g;s/\#baseurl=/baseurl=/g;s/mirrorlist=/#mirrorlist=/g' /etc/yum.repos.d/CentOS*.repo
+        sudo yum install -y epel-release
+        sudo yum -y update
```

Comment:
after this step all your changes mirror->vault might be get back

Reply:
fixed in new commit.